Web_Development/FrontEnd/CSS_Text_Effects/text_effects.md

```diff
@@ -0,0 +1,285 @@
+# CSS Text Effects
+
+There are four components in it: 
+1. CSS Text Overflow
+2. Word Wrap
+3. Line Breaking Rules
+4. Writing Modes
+
+Lets dive into each one!
+
+## 1. CSS Text Overflow
+
+It is used to signal the overflowed content to the user which is hidden from the view.
+
+It doesn't work on its own.<b>white-space: nowrap;</b> and <b>overflow: hidden;</b> has to be used with this property.
+
+### Syntax:
+
+```
+text-overflow: clip|ellipis|initial|string|inherit;
+```
+
+Here the default value is <b>clip</b>.
+
+<b>clip: </b>Clips the overflowed text.
+
+<b>ellipsis: </b>Displays an ellipsis (…) or three dots to show the clipped text.
+
+<b>initial: </b>Iniializes CSS property to its default value.
+
+<b>string: </b>Displays clipped text to the user using the string of programmer's choice. But this feature is only accessible in Firefox browser.
+
+<b>inherit: </b>It will set the text-overflow value to its parent element's value.
+
```

Comment:
If you can write this in a proper way, maybe numbering or use ``` ?

Reply:
sure, will do that.